File: CH3-desicion-sructure-and-boolean-logic/EX9-time-calculator/program-code/timeCalculator.java
import java.util.Scanner;

public class timeCalculator {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int second;
        int result;

        System.out.print("Enter a second :");
        second = input.nextInt();

        if (second < 60) {
            result = second;
            System.out.println(result + "second");
        } else {
            if (second >= 60 && second < 3600) {
                result = second / 60;
                System.out.println(result + "Minute");
            } else {
                if (second >= 3600 && second < 8600) {
                    result = second / 3600;
                    System.out.println(result + "Hour");
                } else {
                    result = second / 86400;
                    System.out.println(result + "Day");
                }
            }
        }

        input.close();
    }

}

File: CH3-desicion-sructure-and-boolean-logic/EX9-time-calculator/program-code/timeCalculator.py
second = input("Enter a second :")

if second < 60:
    result = second
    print(result + " Second")
else:
    if second >= 60 and second < 3600:
        result = second / 60
        print(result + " Minute")
    else:
        if second >= 3600 and second < 86400:
            result = second / 3600
            print(result + "Hour")
        else:
            result = second / 86400
            print(result + "Day")
